client/signup/jetpack-connect/site-url-input.jsx

```diff
@@ -44,41 +49,40 @@ class JetpackConnectSiteURLInput extends Component {
 		this.refs.siteUrl.refs.textField.focus();
```

Comment:
Thanks! I'm not really sure why the ref is even being used here, it doesn't seem like a ref use case. I'm going to see if I can get rid of it completely.